app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+
+class UserController extends Controller
+{
+    public function index()
+    {
+        return view('admin.userList')->with('users',  User::orderBy('id', 'DESC')->paginate(20));
+    }
+
+    public function delete(User $user)
+    {
+        $user->delete();
+        return redirect()->action('UserController@index', ['users' => User::orderBy('id', 'DESC')->paginate(20)])->with('Success', 'The user is deleted successfully.');
```

Comment:
i think no need to declare again user data if you will just redirect to index page, index method already do the job of displaying the list users.